lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -292,27 +292,21 @@ namespace {
 class ADContext;
 
 /// The invoker of a differentiation task. It can be some user syntax, e.g.
-/// `AutoDiffFunctionExpr` expression, the differentiation pass, or nothing at
-/// all. This will be used to emit informative diagnostics.
+/// an `autodiff_function` instruction lowered from an `AutoDiffFunctionExpr`
+/// expression, the differentiation pass, or nothing at all. This will be used
```

Comment:
I didn't change the original description, but I'm curious about what's meant by "nothing at all".
Is that simply referring to arbitrary (maybe handwritten) `autodiff_function` instructions in SIL? Could the description perhaps be improved (e.g. `"or anything at all"`).

cc @rxwei